src/caching.js

```diff
@@ -160,6 +170,8 @@
     }
   });
 
-  RemoteStorage.Caching._rs_init = function () {};
+  RemoteStorage.Caching._rs_init = function (remoteStorage) {
+    this._remoteStorage = remoteStorage;
```

Comment:
Unfortunately, loading/startup process is currently a bit inconsistent and (imo) hacky. In the same file, there's [a note](https://github.com/remotestorage/remotestorage.js/blob/master/src/caching.js#L147-L151) about how the instance is not available yet and that this should be changed eventually. I think @galfert and I added that last time we tried to understand it from just reading the code.

To be honest, I don't know much about it from the top off my head, because I have to re-read all of it every time in order to remember. :/